src/components/CurrentUser/CurrentUser.jsx

```diff
@@ -1,12 +1,44 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import './CurrentUser.scss';
+import PropTypes from 'prop-types';
+import { getUser } from '../../api/api';
 
-export const CurrentUser = () => (
-  <div className="CurrentUser">
-    <h2 className="CurrentUser__title"><span>Selected user: 2</span></h2>
+export const CurrentUser = ({ userId, userClear }) => {
+  const [selectUser, setSelectUser] = useState({});
 
-    <h3 className="CurrentUser__name">Ervin Howell</h3>
-    <p className="CurrentUser__email">Shanna@melissa.tv</p>
-    <p className="CurrentUser__phone">010-692-6593 x09125</p>
-  </div>
-);
+  useEffect(() => {
+    debugger;
+    getUser(userId)
+      .then(users => setSelectUser(users.data));
```

Comment:
```suggestion
      .then(user => setSelectUser(users.data));
```
you should receive only one object